openedx/openedx-platform
# [BB-6041] Convert password compliance warning to html

## Description

This PR converts the password compliance warning message back to HTML. It was converted to string as part of additional security but that resulted in HTML attributes not being applied while displaying the warning message.

**Jira tickets:** [BB-6041](https://tasks.opencraft.com/browse/BB-6041)

## Screenshots:

Before fix:
![esme-password-warning](https://user-images.githubusercontent.com/13742492/160234115-70db7765-5c43-42bb-9f3a-b135a320a91d.png)


After fix:
![esme-stage-warning-after-fix](https://user-images.githubusercontent.com/13742492/160234126-989e9dc5-fbf2-49cb-9e03-e39ff12f8fed.png)


## Testing instructions

1. Checkout this branch and setup a minimum password length validator, enable compliance on login and set a deadline.
2. Register or login with a password that doesn't meet the password requirement.
3. Verify the warning message shown is formatted correctly.

**Reviewers:**

TBD
